yucca/image_processing/transforms/BiasField.py

```diff
@@ -3,16 +3,25 @@
 
 
 class BiasField(YuccaTransform):
-    def __init__(self, data_key="image", p_per_sample=1):
+    def __init__(
+        self,
+        data_key="image",
+        p_per_sample=1,
+        clip_to_input_range=False,
+    ):
         self.data_key = data_key
         self.p_per_sample = p_per_sample
+        self.clip_to_input_range = clip_to_input_range
 
     @staticmethod
     def get_params():
         # No parameters to retrieve
         pass
 
     def __biasField__(self, imageVolume):
+        mn = imageVolume.min()
+        mx = imageVolume.max()
+
         if len(imageVolume.shape) == 3:
```

Comment:
I know this is not from now, but to me we should never do camelCase for variables. In this file we are currently combining camelCase and snake_case. I would change this to `vol`, `image` `x`, `image_volume` or similar.